agent/structs/structs.deepcopy.go

```diff
@@ -18,6 +18,14 @@ func (o *APIGatewayListener) DeepCopy() *APIGatewayListener {
 		cp.TLS.CipherSuites = make([]types.TLSCipherSuite, len(o.TLS.CipherSuites))
 		copy(cp.TLS.CipherSuites, o.TLS.CipherSuites)
 	}
+	if o.Override.JWT != nil {
```

Comment:
so for this I'm hoping that my PR for https://github.com/globusdigital/deep-copy/pull/33 gets merged which will let us generate `DeepCopy` methods with build tags for both ent and oss without conflict

Reply:
if this doesn't I'll hand write the deep copy methods for now

Reply:
manually wrote the deep copy methods for the time being, if that PR gets merged I'll update this to use the generated methods with build tags